.github/workflows/all_plugins.yaml

```diff
@@ -42,12 +40,12 @@ jobs:
     timeout-minutes: 30
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: 'Install Flutter'
-        run: ./.github/workflows/scripts/install-flutter.sh stable
-      - name: 'Install Tools'
-        run: |
-          ./.github/workflows/scripts/install-tools.sh
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
```

Comment:
GitHub recommends because of security using a full hash reference for actions:

> By full hash changeset reference: uses: owner/action-name@26968a09c0ea4f3e233fdddbafd1166051a095f6. Even though [SHA-1 collisions were proven successful](https://shattered.io/) some time ago, [preimage attacks](https://en.wikipedia.org/wiki/Preimage_attack) against a Git object hash are [more difficult to achieve](https://github.blog/2017-03-20-sha-1-collision-detection-on-github-com/). Currently, this is the safest way to reference a specific snapshot of an action.

https://securitylab.github.com/research/github-actions-building-blocks/

I think it would be a good idea to use a hash reference instead of a tag reference. What do you think?

Reply:
Hum, I think I would rather have the latest version of the action. There are no secrets involved when we use this action and no action on the repository so I think it's ok